Integrations/SplunkPy/CHANGELOG.md

```diff
@@ -21,4 +21,4 @@ Added the *Earliest time to fetch* and *Latest time to fetch* parameters, which
 -
 
 ## [19.9.0] - 2019-09-04
-- Added the *Fetch limit* parameter to the instance configuration, which specified the maximum number of results to fetch.
+- Added the *Fetch limit* parameter to the instance configuration, which specified the maximum number of results to fetch.
```

Comment:
I don't even know why this is different...

Reply:
master has empty line at the end (don't know why it doesn't show here...)